<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
    <line>
      <![CDATA[Mozilla is envoked]]>
    </line>
    <line>
      <![CDATA[URL is http://111.125.141.74:8086/VaticaHealth.Web.UI/]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="56514" started-at="2017-01-09T08:06:57Z" finished-at="2017-01-09T08:07:54Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="56514" started-at="2017-01-09T08:06:57Z" finished-at="2017-01-09T08:07:54Z">
      <class name="com.vaticahealth.vatica.testsuits.SmokeTest2">
        <test-method status="PASS" signature="invokeUrl()[pri:0, instance:com.vaticahealth.vatica.testsuits.SmokeTest2@7de62196]" name="invokeUrl" is-config="true" duration-ms="3709" started-at="2017-01-09T13:36:57Z" finished-at="2017-01-09T13:37:01Z">
          <reporter-output>
            <line>
              <![CDATA[URL is http://111.125.141.74:8086/VaticaHealth.Web.UI/]]>
            </line>
          </reporter-output>
        </test-method> <!-- invokeUrl -->
        <test-method status="FAIL" signature="test1()[pri:0, instance:com.vaticahealth.vatica.testsuits.SmokeTest2@7de62196]" name="test1" duration-ms="45433" started-at="2017-01-09T13:37:01Z" finished-at="2017-01-09T13:37:46Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[PPP header doesn't match expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: PPP header doesn't match expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.vaticahealth.vatica.testcases.PPPTest.verifyHeaderOf_PPPTab(PPPTest.java:19)
	at com.vaticahealth.vatica.testsuits.SmokeTest2.test1(SmokeTest2.java:25)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method status="PASS" signature="QuitDriver()[pri:0, instance:com.vaticahealth.vatica.testsuits.SmokeTest2@7de62196]" name="QuitDriver" duration-ms="7346" started-at="2017-01-09T13:37:46Z" finished-at="2017-01-09T13:37:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- QuitDriver -->
      </class> <!-- com.vaticahealth.vatica.testsuits.SmokeTest2 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
